tests/PSR7/Validators/BodyValidatorTest.php

```diff
@@ -76,7 +84,7 @@ public function testValidateGreen(string $specFile, string $message) : void
         );
 
         $validator = (new ValidatorBuilder())->fromYamlFile($specFile)->getServerRequestValidator();
-        $validator->validate($serverRequest);
-        $this->addToAssertionCount(1);
+        $opAddress = $validator->validate($serverRequest);
+        $this->assertEquals($expectedOpAddress, $opAddress);
```

Comment:
Do we really need this assertion? In case we have only one op addr in fixture schema not catching an exception here means that we matched that single one